code/modules/antagonists/heretic/magic/burglar_finesse.dm

```diff
@@ -0,0 +1,33 @@
+/datum/action/cooldown/spell/pointed/burglar_finesse
+	name = "Burglar's Finesse"
+	desc = "Steal a random item from the victims backpack, or any other storage item if not found."
+	background_icon_state = "bg_heretic"
+	overlay_icon_state = "bg_heretic_border"
+	button_icon = 'icons/mob/actions/actions_ecult.dmi'
+	button_icon_state = "burglarsfinesse"
+
+	school = SCHOOL_FORBIDDEN
+	cooldown_time = 40 SECONDS
+
+	invocation = "Y'O'K!"
+	invocation_type = INVOCATION_WHISPER
+	spell_requirements = NONE
+
+	cast_range = 4
+
+/datum/action/cooldown/spell/pointed/burglar_finesse/is_valid_target(atom/cast_on)
+	return ..() && ishuman(cast_on) && (locate(/obj/item/storage) in cast_on.contents)
```

Comment:
This should probably be checking for atoms with an `atom_storage`, rather than just `/obj/item/storage` items.

Reply:
atom_storage includes storage implants